components/brave_rewards/core/wallet_provider/connect_external_wallet.cc

```diff
@@ -177,18 +177,27 @@ void ConnectExternalWallet::OnConnect(
     return std::move(callback).Run(ConnectExternalWalletResult::kUnexpected);
   }
 
-  from_status == mojom::WalletStatus::kNotConnected
-      ? engine_->client()->ExternalWalletConnected()
-      : engine_->client()->ExternalWalletReconnected();
-  engine_->database()->SaveEventLog(
-      log::kWalletVerified,
-      WalletType() + std::string("/") + abbreviated_address);
+  // Set the "active" wallet type.
+  engine_->SetState(state::kExternalWalletType, std::string(WalletType()));
 
   // Update the user's "declared country" based on the information provided by
   // the linking endpoint.
   CHECK(result.has_value() && !result.value().empty());
   engine_->SetState(state::kDeclaredGeo, result.value());
 
+  // Attempt to transfer any unspent vBAT to the user's external wallet.
+  engine_->promotion()->TransferTokens(base::DoNothing());
```

Comment:
Moved from the "get wallet" flow. @szilardszaloki Does it seem fine to move it here (to the connect wallet flow), or was there a particular reason for putting it in the "get wallet" flow?

Reply:
@zenparsing The reason it was in the "get wallet" flow is that this flow runs more frequently than any other flows.

Reply:
The transfer endpoint is no longer supported, so we'll remove the call.